addons/website_sale/views/templates.xml

```diff
@@ -1013,7 +1013,7 @@
                                     </t>
                                 </t>
                                 <t t-else="">
-                                    <span class="text-muted w-100" t-esc="int(line.product_uom_qty)"/>
+                                    <input type="text" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty" />
```

Comment:
You can rebase, this is merged with https://github.com/odoo/odoo/commit/b9d157e7c6ea53674f4d24b4207ae934e7f19c3e